tests/Auth/LoginTest.php

```diff
@@ -67,6 +70,9 @@ public function testLoginSuccessAdmin()
 
     public function testAllowRememberShowsUp()
     {
+        // primitive perhaps, but works: make sure we are logged out first:
+        $response = $this->get(route_to('logout'));
```

Comment:
This works but could always be added to the `setUp()` method of the tests so it's only in one location.